### ISSUE TITLE
vmagent: panic when using the same topic name with multiple kafka brokers 

### ISSUE THREAD

**Issue by zekker6** [VictoriaMetrics/VictoriaMetrics#6636]
### Describe the bug

There are 2 kafka brokers which receive data for a specific environment:
- `kafka-a`
- `kafka-b`
Both brokers are using the same naming for topic, let's say `topic-a`.

### To Reproduce

Run vmagent with the following flags:
```
-kafka.consumer.topic.brokers=kafka-a,kafka-b
-kafka.consumer.topic.groupID=example-group,example-group
-kafka.consumer.topic=topic-a,topic-a
-kafka.consumer.topic.format: jsonline,jsonline
```

This will result in:
```
panic: BUG: metric "vmagent_kafka_messages_read_total{topic=\"topic-a\",format=\"jsonline\",worker_id=\"0\"}" is already registered
```

### Version

v1.101.0

**Comment by valyala** [comment]
FYI, this issue has been fixed in [v1.97.6 LTS release](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.97.6). The bugfix will be included in the upcoming release too.

**Comment by valyala** [comment]
This issue has been fixed in [v1.102.0 release](https://docs.victoriametrics.com/changelog/).
